src/cljs/nr/cardbrowser.cljs

```diff
@@ -120,13 +120,12 @@
 
 (defn- expand-one
   "Reducer function to create a previous card from a newer card definition."
-  [acc version c]
-  (let [number (str->int (subs version 3))
-        cycle-pos (str->int (subs version 0 2))
-        prev-set (find-first #(= cycle-pos (:cycle_position %)) @cards/sets)
+  [acc {:keys [code set_code] :as version} c]
```

Comment:
this should be kebab-case, right?

Reply:
You might think that, but it is actually underscore in the data.
```
web.core=> (@jinteki.cards/all-cards "Swift")
{:faction "Criminal", :_id "60745d0fd8161e6bee8fb62e", :format {:standard "legal", :system-gateway "rotated", :startup "legal", :eternal "legal", :snapshot "rotated", :snapshot-plus "rotated", :classic "legal"}, :rotated false, :cycle_code "ashes", :factioncost 2, :setname "Uprising Booster Pack", :number 2, :images {:en {:high {:26078 "/img/cards/en/high/stock/26078.png"}, :default {:stock "/img/cards/en/default/stock/27002.png", :26078 "/img/cards/en/default/stock/26078.png"}}, :zh-simp {:default {:stock "/img/cards/zh-simp/default/stock/27002.png", :26078 "/img/cards/zh-simp/default/stock/26078.png"}}}, :type "Hardware", :title "Swift", :uniqueness true, :normalizedtitle "swift", :code "27002", :side "Runner", :cost 2, :quantity 3, :deck-limit 3, 
   :set_code "urbp",
   :previous-versions [{:code "26078", :set_code "ur"}], 
:subtype "Console - Vehicle", :text "+1[mu]\nThe first time each turn you play a <strong>run</strong> event, gain [click].\nLimit 1 <strong>console</strong> per player.", :subtypes ["Console" "Vehicle"]}
```

Reply:
Same with `cycle_code`

Reply:
Huh! Okay. Cool!